### ISSUE TITLE
Publish jtflib to CDN?

### ISSUE THREAD

**Issue by situx** [cdli-gh/jtf-lib#3]
Would it be possible to provide a version of your library on a public CDN?
This would simplify reusage of the library to validate ATF and to convert ATF to JTF and vice versa

**Comment by sengarsameer** [comment]
jsdelivr is good, we can also think about cdnjs [https://cdnjs.com/about](https://cdnjs.com/about). They have a good market share and userbase.

**Comment by larsgw** [comment]
Do you mean to reuse it in the browser?

**Comment by situx** [comment]
Yes, that was my idea. Currently, you have to build the library using npm and browserify and it would be easier if you could access the library from a CDN.

**Comment by larsgw** [comment]
So what we need to do is make a browserify (or webpack) build and put the resulting bundle on GitHub or npm because jsdelivr and unpkg already serve those files anyway (unpkg only for npm).

**Comment by situx** [comment]
I believe so
